src/main/java/data/CameraTimeline.java

```diff
@@ -12,6 +14,9 @@
     // The camera that is associated with this timeline.
     @Getter @Setter
     private Camera camera;
+    
+    @Getter
+    private ArrayList<CameraShot> shots;
```

Comment:
Oke please delete the list of shots from the TimLine class then, that is not used anymore